user_docs/de/userGuide.t2t

```diff
@@ -2762,10 +2762,6 @@ Folgende Braillezeilen werden unterstützt:
 
 Diese Braillezeilen unterstützt nicht die automatische Braillezeilenerkennung im Hintergrund von NVDA.
 
-Falls BrxCom installiert ist, wird es benutzt.

```

Comment:
Could you remove the changes from this file?
The German translation will need to be fixed by our translators via the translation system: https://github.com/nvaccess/nvda/wiki/TranslatingUserGuide

Reply:
should be restored!